### ISSUE TITLE
Old swagger-jsdoc package causing files to not be found

### ISSUE THREAD

**Issue by brandonmpetty** [patsimm/swagger-jsdoc-webpack-plugin#2]
When I ran this in my node project it could never find any of my source files to scan for documentation.
That is when I realized you are running v3.  I updated the package:

```
  "dependencies": {
    "swagger-jsdoc": "^6.1.0"
  }
```

That fixed everything.
Also, with the newer version, some of the options may have changed so you may want to update the readme.

Example:

```
{
    definition: {
        openapi: '3.0.0',
        info: {
            title: 'Hello World',
            version: '1.0.0',
        },
    },
    apis: ['./src/**/*.js'],
}
```

Where **swaggerDefinition** is now **definition**.

**Comment by patsimm** [comment]
Thanks @brandonmpetty for your Issue. First I want to apolgize for my late reply. I will try to have a look into this later today :)

**Comment by patsimm** [comment]
I updated the dependency. And released version 2.0.0 of this package.
